src/components/new-request/name-check/name-check.vue

```diff
@@ -634,7 +661,7 @@ export default class NameCheck extends Vue {
       return { color: 'red darken-2', icon: 'mdi-alert-octagon' }
     }
     if (this.hasIssuesConflictCaution || this.hasIssuesStructureCaution) {
-      return { color: 'orange darken-2', icon: 'mdi-alert' }
+      return { color: '#F8661A', icon: 'mdi-alert' }
```

Comment:
This hex code is already defined as `$caution`, but that's not a Vuetify theme color... please consider adding it (see vuetify.ts), or maybe replacing `warning` and then using that name here.

(It's bad practice to use hex colors in code, and good practice to have a defined palette for an app.)

Reply:
ah okay thanks, I was wondering about this. I will replace 'warning'

Reply:
I ended up creating a new theme: 'caution' because warning is used for the banner